aws/notebook/Dockerfile

```diff
@@ -0,0 +1,63 @@
+FROM jupyter/base-notebook
+
+USER root
+RUN apt-get update \
+  && apt-get install -yq --no-install-recommends libfuse-dev nano fuse \
+  && apt-get clean \
+  && rm -rf /var/lib/apt/lists/*
+USER $NB_USER
+
+RUN conda install --yes -c conda-forge \
+    cytoolz \
+    dask \
+    distributed \
+    fastparquet \
+    git \
+    ipywidgets \
+    jupyterlab \
+    matplotlib \
+    nb_conda_kernels \
+    netcdf4 \
+    nomkl \
+    numba \
+    numcodecs \
+    numpy \
+    pandas \
+    python-blosc \
+    scipy \
+    xarray \
+    zict \
+    && conda clean -tipsy
+
+RUN jupyter labextension install @jupyter-widgets/jupyterlab-manager
+
+RUN pip install fusepy click jedi kubernetes --upgrade --no-cache-dir
+
+RUN pip install git+https://github.com/mrocklin/daskernetes@deploy \
+                git+https://github.com/zarr-developers/zarr \
+                git+https://github.com/jhamman/xarray@fix/zarr_set_attrs \
+                git+https://github.com/martindurant/gcsfs.git@fuse \
+                git+https://github.com/dask/distributed@master \
+                git+https://github.com/dask/dask@master \
+                --no-cache-dir
```

Comment:
More generally, we can probably move to full releases / conda installs of a bunch of these.

Reply:
Is it possible to separate the python environment specification into its own file (environment.yml)? Or does something about docker require the whole DOCKERFILE to be a single self-contained script. Separating the python environment stuff from the more general setup could make it easier to define new environments down the line...

This is partly related to #87.